trailingstoploss_tp.py

```diff
@@ -318,6 +356,8 @@ def handle_new_deal(thebot, deal, profit_config):
 
     activation_percentage = float(profit_config.get("activation-percentage"))
 
+    sl_timeout = float(profit_config.get("sl-timeout"))
```

Comment:
Waarom hier geen cast naar int, zodat je dat onderin bij de logregel niet meer hoeft te doen? Nu is het twee keer een cast, beetje zonde.